tests/integration/tls_tests/test_tls.py

```diff
@@ -27,11 +27,16 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
     async with ops_test.fast_forward():
         my_charm = await ops_test.build_charm(".")
         resources = {"mongodb-image": METADATA["resources"]["mongodb-image"]["upstream-source"]}
-        await ops_test.model.deploy(my_charm, num_units=3, resources=resources)
+        await ops_test.model.deploy(my_charm, num_units=3, resources=resources, series="jammy")
         await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status="active", timeout=2000)
 
         config = {"generate-self-signed-certificates": "true", "ca-common-name": "Test CA"}
-        await ops_test.model.deploy(TLS_CERTIFICATES_APP_NAME, channel="beta", config=config)
+        await ops_test.model.deploy(
+            TLS_CERTIFICATES_APP_NAME,
+            channel="beta",
+            config=config,
+            series="focal",
```

Comment:
I assume that TLS-certs isn't availbe on jammy yet

Reply:
you right, tls-certificates-operator is on focal for now